statistical_tests/statistical_tests.md

```diff
@@ -2,13 +2,15 @@
 
 author:   Rose Hartman
 email:    hartmanr1@chop.edu
-version:  1.0.0
+version:  1.1.0
```

Comment:
I wonder if this would even be considered version 1.0.1? Seems like most of the changes are behind the scenes, though that one additional external resource link might bump it up...

Reply:
You're probably right! I changed that at the beginning because I was anticipating much more substantial updating, like I had to do for ggplot2, but this is almost all on the order of fixing broken links. I'll revise to 1.0.1